06.build-guide/01.basic-build-guide/docs.md

```diff
@@ -15,47 +15,40 @@ This build guide contains a set of general instructions for building High Fideli
 - [OpenSSL](https://www.openssl.org/): Use the latest available 1.0 version (**NOT 1.1**) of OpenSSL to avoid security vulnerabilities.
-- [VHACD](https://github.com/virneo/v-hacd)(clone this repository)(Optional)
+- [VHACD](https://github.com/virneo/v-hacd): Clone this repository (Optional)
 
-### CMake External Project Dependencies
+## CMake External Project Dependencies
 
-These dependencies need not be installed manually. They are automatically downloaded on the platforms where they are required.  
+These dependencies do not need to be installed manually. They are automatically downloaded on the platforms where they are required. 
+ 
 - [Bullet Physics Engine](https://github.com/bulletphysics/bullet3/releases):  2.83
 - [GLEW](http://glew.sourceforge.net/):   1.13
 - [glm](https://glm.g-truc.net/0.9.8/index.html):  0.9.8
-- [gverb](https://github.com/highfidelity/gverb)
 - [Oculus SDK](https://developer.oculus.com/downloads/):   1.11 (Win32) / 0.5 (Mac)
 - [OpenVR](https://github.com/ValveSoftware/openvr):   1.0.6 (Win32 only)
 - [Polyvox](http://www.volumesoffun.com/):   0.2.1
 - [QuaZip](https://sourceforge.net/projects/quazip/files/quazip/):   0.7.3
 - [SDL2](https://www.libsdl.org/download-2.0.php):   2.0.3
-- [soxr](https://sourceforge.net/p/soxr/wiki/Home/):   0.1.1
 - [Intel Threading Building Blocks](https://www.threadingbuildingblocks.org/):   4.3
 - [Sixense](http://sixense.com/):   071615
 - [zlib](http://www.zlib.net/):   1.28 (Win32 only)
 - nVidia Texture Tools:   2.1
 
-### OS Specific Build Guides
+The above dependencies will be downloaded, built, linked and included automatically by CMake where we require them. The CMakeLists files that handle grabbing each of the following external dependencies can be found in the [cmake/externals folder](https://github.com/highfidelity/hifi/tree/master/cmake/externals). The resulting downloads, source files and binaries will be placed in the `build/ext` folder in each of the subfolders for each external project.
 
-- [Linux](../linux)
-- [OS X](https://docs.highfidelity.com/build-guide/mac-os)
-- [Windows 64-bit](../windows-64-bit)
-- [Android](../android)
+These are not placed in your normal build tree when doing an out of source build so that they do not need to be re-downloaded and re-compiled every time the CMake build folder is cleared. Should you want to force a re-download and re-compile of a specific external, you can simply remove that directory from the appropriate subfolder in `build/ext`. Should you want to force a re-download and re-compile of all externals, just remove the `build/ext` folder.
+
+If you would like to use a specific install of a dependency instead of the version that would be grabbed as a CMake ExternalProject, you can pass -DUSE_LOCAL_$NAME=0 (where $NAME is the name of the subfolder in [cmake/externals](https://github.com/highfidelity/hifi/tree/master/cmake/externals)) when you run CMake to tell it not to get that dependency as an external project.
```

Comment:
I'm not sure this `USE_LOCAL_` mechanism works at all anymore.

Reply:
@jherico Who would I check with to confirm? It is still in https://github.com/highfidelity/hifi/blob/master/BUILD.md where I pulled most of this content from.